services/terrain/project.clj

```diff
@@ -7,13 +7,13 @@
       (string/trim (:out (sh "git" "rev-parse" "HEAD")))
       ""))
 
-(defproject org.iplantc/donkey "5.0.0-SNAPSHOT"
+(defproject org.iplantc/terrain "5.0.0-SNAPSHOT"
   :description "Framework for hosting DiscoveryEnvironment metadata services."
-  :url "https://github.com/iPlantCollaborativeOpenSource/Donkey"
+  :url "https://github.com/iPlantCollaborativeOpenSource/Terrain"
```

Comment:
Maybe point this to the new repo? This seems to be two iterations out of date. I'm happy to look into the other services, since I hadn't even thought about this line